docs/usage.rst

```diff
@@ -116,24 +116,36 @@ It has structure following pytest test item's `nodeid`_.
     - test classes are directories
     - test functions are directories (each parametrized testcase variant is distinct)
     - each registered logger is a file (root logger is called 'root')
+    - if some test fails, additional directory named `failedlogs` is created in top-level logs directory. It wil contain the same exact directory tree as the original one, and a symbolic link at the end (pointing th the original directory with logs from that testcase)
 
 ::
 
     logs/
     ├── classtests
     │   └── test_y.py
     │       └── TestClass
-    │           └── test_class
-    │               ├── daemon
-    │               └── setup
+    │           ├── test_class
+    │           |   ├── daemon
+    │           |   └── setup
+    |           └── test_that_failed_two
+    |               └── somelogfile
+    ├── failedlogs
```

Comment:
Ok, I see your point. I agree that #1 seems like a better way. 
I would still want to be able to make such 'failedlogs' dir inside the main logsdir - as it simplifies log collection (eg. on CI - as It would only require to collect single directory content and all would be inside). How do you think this should be handled? Single option accepting some complex path (eg logs/failedlogs), or a separate option?

Reply:
Hmm, currently we have this `logs` directory under tmpdir path:
`/tmp/pytest-of-aurzenligl/pytest-137/logs/`
I think it's reasonable to keep its contents/name/location intact, without any additional directories inside. Which assumes we'd need to produce extra (sibling) directory, like:
`/tmp/pytest-of-aurzenligl/pytest-137/logs-failed/`
with symlinks inside. Then CI could transfer `.../pytest-137/logs*` paths as build artifacts. If someone is interested in `logs` as such, you'd read one dir, if you're interested in `logs-failed` - you have second dir for that.

I think `logs-failed` could be generated without explicit user configuration (it doesn't seem to hurt anyone), but if user would like to see a link to `*/pytest-137/logs-failed/` directly in test suite dir (e.g. next to `conftest.py`), I think explicit configuration (preferably via `pytest_logger_config` hook) would be needed.

Reply:
> I think logs-failed could be generated without explicit user configuration (it doesn't seem to hurt anyone)

Well. It doesn't break interface but can be misleading for people who are not familiar with `pytest-logger`. I mean that if we have two directories with logs: `failed-logs` and `logs` one may assume that the second one contains `passed-logs`. Extracting just failing introduces asymmetry to the interface.

In order to avoid such problem I'd consider adding something like
```
logger_config.split_logs_by_result(('passed', 'failed'))
```
If one wants to have a single directory with everything embedded he can use it with `('logs/passed', 'logs/failed')`.

Reply:
I like @emdej 's proposition better than mine. However it comes with extra difficulty - test status (`passed`/`failed`/`skipped`) is determined after logs have been produced (true even in `skipped` case).

Reply:
Ok, I will do some config.

Reply:
ok, I made it configurable. please check

Reply:
Thank you -> I'll review during weekend in the worst case.